src/main.js

```diff
@@ -0,0 +1,25 @@
+var buttonMenu = document.querySelector('.menu-open');
+var menuOpen = document.querySelector('.open-menu');
+var menuClose = document.querySelector('.menu-close');
+
+buttonMenu.addEventListener('click', toggleMenu);
+menuClose.addEventListener('click', displayBack);
+
+
+function toggleMenu() {
+  displayOff();
+  menuClose.classList.remove('hidden');
+  menuOpen.classList.remove('hidden');
+}
+
+function displayBack() {
+  displayOff();
+  buttonMenu.classList.remove('hidden');
+}
+
+
+function displayOff() {
```

Comment:
I really like how you created this function and invoked it in two other functions in an attempt to make your code more DRY.  There is probably a way to refactor some of these functions so that an argument (variable) can be passed into a parameter, since we are removing .hidden multiple times.